data_structures/arrays/remove_vowels.rb

```diff
@@ -0,0 +1,74 @@
+# Challenge name: Remove vowels from a string
+#
+# Given a string s, remove the vowels 'a', 'e', 'i', 'o', and 'u'
+# from it, and return the new string.
+# Example 1:
+# Input: s = "leetcodeisacommunityforcoders"
+# Output: "ltcdscmmntyfrcdrs"
+#
+# Example 2:
+# Input: s = "aeiou"
+# Output: ""
+#
+# @param {String} s
+# @return {String}
+
+#
+# Approach 1: Brute Force
+#
+# Time Complexity: O(n)
+#
+def remove_vowels(s)
+  result_array = []
+  s.downcase
```

Comment:
```suggestion
  s.downcase!
```

Did you mean `downcase!`?